src/css/typography.css

```diff
@@ -19,11 +19,12 @@
     @apply font-bold mt-6 mb-2;
   }
 
-  p {
+  p, ul {
     @apply text-lg;
   }
+
   @screen md {
-    p {
+    p, ul {
       @apply text-xl;
```

Comment:
Lots of love for `@apply` over class cluttering

Reply:
You mean to abstract away the tailwind classes into named classes?

Reply:
Named classes or just selectors like these, keeping it away from the HTML is 🔥

Reply:
Unfortunately, it breaks most of the point and efficiencies of a utility framework though.

Reply:
https://twitter.com/adamwathan/status/1226511611592085504

Reply:
That his opinion, which I absolutely disagree with. HTML describes the content, not the appearance. A hill I'll happily die on. :D